src/wazuh_modules/wm_database.c

```diff
@@ -76,11 +76,22 @@ static void wm_sync_manager();
 static void wm_check_agents();
 
 /**
- * @brief Method to synchronize 'client.keys' and 'global.db'. All new agents found in 'client.keys will be added to the DB
- *        and any agent in the DB that doesn't have a key will be removed.
+ * @brief Method to synchronize 'client.keys' and 'global.db'. All new agents found
+ *        in 'client.keys will be added to the DB and any agent in the DB that doesn't
+ *        have a key will be removed.
+ *        This method will also create and remove the agents artifacts acorting to
+ *        the action taken in the database with the agent.
  */
 static void wm_sync_agents();
 
+/**
+ * @brief Method to synchronize the agent artifacts with 'client.keys' and 'global.db'.
+ *        For all new agents found in 'client.keys will be created its artifacts.
```

Comment:
Maybe we could reword the sentence: `For all new agents found in 'client.keys will be created its artifacts.`

Reply:
Fixed